docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+services:
+  frozen_flame:
+    container_name: frozenFlame
+    build: .
```

Comment:
This shouldn't be required for a base docker-compose file as it would be built from github actions

Reply:
This is the version for people who want use the github version. The docker-compose for people who want use the image from the docker hub will be propose in the README.MD